core/modules/comment/comment.module

```diff
@@ -1479,21 +1471,17 @@ function comment_config_create_validate(Config $staging_config, $all_changes) {
  * replied to. This prevents people from changing or revising their statements
  * based on the replies to their posts.
  *
- * @param $op
+ * @param string $op
  *   The operation that is to be performed on the comment. Only 'edit' is
  *   recognized now.
```

Comment:
This docblock description for $op also needs to be updated. If I've read this PR right, `'edit'` is in fact not supported at all and this supports many other operations now.

Because this is an API expansion, we should add a `@since` docblock along the following lines:

```
@since 1.10.0 Added x, y, z as $op options.
@since 1.10.0 The "edit" $op option is deprecated. Use "update" instead.
```

Reply:
Fixed in https://github.com/backdrop/backdrop/pull/2451